easybuild/easyconfigs/m/McStas/McStas-3.5.24-foss-2023a.eb

```diff
@@ -0,0 +1,46 @@
+# Author: Erik B Knudsen
```

Comment:
Unless there is a particular reason to keep it we aim to have one version of software per TC.
Can you remove the older one if the new one is the most relevant one?